core/pkg/artifacts/downloader.go

```diff
@@ -120,6 +121,9 @@ func (ad *ArtifactDownloader) downloadFiles(artifactID string, manifest Manifest
 			cursor = response.Artifact.Files.PageInfo.EndCursor
 			for _, edge := range response.GetArtifact().GetFiles().Edges {
 				filePath := edge.GetNode().Name
+				if !strings.HasPrefix(filePath, ad.PathPrefix) {
```

Comment:
Is the behavior correct here? This will count "some/art" as a prefix of "some/artifact".